packages/demo-cards/claim.gts

```diff
@@ -91,6 +119,9 @@ class Isolated extends Component<typeof Claim> {
       return -1;
     }
   }
+  claim = async () => {
```

Comment:
technically you should follow the same pattern of having your button call an `@action` which in turn calls a perform() on an ember concurrency task (and optionally a loading indicator in the template for your claim task).

Reply:
resolved https://github.com/cardstack/boxel/pull/353/commits/7f13eca41c2bc052f581d29ff821d3969a86575b